### PR TITLE
Feat/add  0auth

### PR DESCRIPTION
**Pull request by santaLucas**

### CODE REVIEW

File: public/locales/ru/translation.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "Map": "",
   "Leave Ukraine": "Выехать из Украины",
   "Border info": "Информация о границе",
+  "Login": "логин",
```
Comment (plotka) [inline public/locales/ru/translation.json:5]: In russian "Войти" is commonly used for the login button text

```suggestion
  "Login": "Войти",
```

File: public/locales/it/translation.json [hunk 2]
```diff
@@ -2,6 +2,8 @@
   "Map": "",
   "Leave Ukraine": "Lascia l'Ucraina",
   "Border info": "Informazioni sulla frontiera",
+  "Login": "Accedi",
+  "Logout": "Disconnettersi",
```
Comment (Jystro) [inline public/locales/it/translation.json:6]:
```suggestion
  "Logout": "Disconnettiti",
```

**Review by Klarkin2 — APPROVED** [review]

File: src/index.js [hunk 3]
```diff
@@ -7,7 +7,28 @@ import "./i18n/i18n";
...
+//           clientId={clientId}
+//           redirectUri={window.location.origin}>
+//           <App />
+//         </Auth0Provider>
+//       </Provider>
+//     </BrowserRouter>
+//   </React.StrictMode>,
+//   document.getElementById("root")
+// );
+// =======
```
Comment (plotka) [inline src/index.js:31]: the comment section can be removed?

Comment (santaLucas) [reply src/index.js:31]: ah mb

**Review by Klarkin2 — APPROVED** [review]

**Pull request merged**